web-animations/interfaces/KeyframeEffect/keyframe-properties.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<meta charset='utf-8'>
+<title>Web Animations API: Keyframe Property tests</title>
+<link rel='help' href='https://w3c.github.io/web-animations/#processing-a-keyframes-argument'>
+<script src='/resources/testharness.js'></script>
+<script src='/resources/testharnessreport.js'></script>
+<script src='../../testcommon.js'></script>
+<script src='../../resources/keyframe-utils.js'></script>
+<body>
+<script>
+'use strict';
+
+test(function(t) {
+  var keyframe = {};
+  Object.defineProperty(keyframe, 'width', {value: '200px'});
+  Object.defineProperty(keyframe, 'height', {
+    value: '100px',
+    enumerable: true});
+  assert_equals(keyframe.width, '200px', 'width of keyframe is readable');
+  assert_equals(keyframe.height, '100px', 'height of keyframe is readable');
+  var anim = createDiv(t).animate([keyframe, {height: '200px'}], 1);
+  assert_frame_lists_equal(anim.effect.getKeyframes(), [
+      {offset: null, computedOffset: 0, easing: 'linear', height: '100px'},
+      {offset: null, computedOffset: 1, easing: 'linear', height: '200px'},
+  ]);
+}, 'Only enumerable properties on keyframes are considered');
```

Comment:
This is good. Do you think it's worth adding a test for when we pass the object directly instead of as an element in an array?